helper/tests/helper-tests-common/src/main/resources/CCDV2.avsc

```diff
@@ -0,0 +1,86 @@
+{
+  "type": "record",
```

Comment:
this schema is "too complex" and also lacks unions maps and lists for corner cases

Reply:
we have the union in the test. will check other cases.